domains/transfer/Transfer.API/API/Cvr/Api/v2024_01_03/Controllers/CvrController.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using API.Cvr.Api.Clients.Cvr;
+using API.Cvr.Api.Models;
+using API.Cvr.Api.v2024_01_03.Dto.Responses;
+using Asp.Versioning;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Cvr.Api.v2024_01_03.Controllers;
+
+[Authorize]
+[ApiController]
+[ApiVersion(20240103)]
+[Route("api/cvr")]
+public class CvrController(CvrClient client) : Controller
+{
+    /// <summary>
+    /// Get CVR registered company information for multiple CVR numbers
+    /// </summary>
+    /// <response code="200">Successful operation</response>
+    /// <response code="404">CVR company not found</response>
+    [ProducesResponseType(typeof(CvrCompanyListResponse), 200)]
+    [ProducesResponseType(typeof(void), 404)]
+    [HttpPost]
+    public async Task<ActionResult<CvrCompanyListResponse>> GetCvrCompanies([FromBody] List<string> cvrNumbers)
+    {
+        var parsedCvrNumbers = cvrNumbers.Select(CvrNumber.TryParse).OfType<CvrNumber>().ToList();
+
+        if (!parsedCvrNumbers.Any())
+            return Ok(new CvrCompanyListResponse(new List<CvrCompanyDto>()));
+
+        var searchResult = await client.CvrNumberSearch(parsedCvrNumbers);
+        if (searchResult?.hits?.hits == null)
```

Comment:
What is returned if the first hits is null?

Reply:
Nothing, the hit is ignored, as if it was never made

Reply:
Will the api return 200 ok empty list?

Reply:
Yes.